app/src/main/java/com/infomaniak/drive/data/api/ApiRoutes.kt

```diff
@@ -118,6 +119,16 @@ object ApiRoutes {
 
     fun shareLink(file: File) = "${fileURL(file)}/link"
 
+    fun createCategory(driveId: Int) = "${DRIVE_API}$driveId/category"
+
+    fun editCategory(driveId: Int, categoryId: Int) = "${DRIVE_API}$driveId/category/$categoryId"
```

Comment:
`editCategory` and `deleteCategory` is the same url, you can merge it, and differentiate them in the repository